ab3d2_source/menu/menunb.s

```diff
@@ -977,7 +993,15 @@ mnu_waitmenu:;out: d0=Selection number
 .sliderl:
 				moveq.l	#42,d1
 				bra.s	.cpcont
-
+.cd32:
+				clr.b	Plr1_Keys_b
+				clr.b	Plr1_Path_b
+				clr.b	Plr1_Mouse_b
+				st		Plr1_Joystick_b
+				clr.b	Plr2_Keys_b
```

Comment:
Do we need to manipulate the Plr2 state here?

Reply:
I do not know. I will try and test it in 2 player later.